localstack/utils/bootstrap.py

```diff
@@ -360,6 +361,56 @@ def start_infra_locally():
     return infra.start_infra()
 
 
+def validate_localstack_config(name):
+    dirname = os.getcwd()
+    compose_file_name = '%s/%s' % (dirname, name)
+
+    # validating docker-compose file
+    cmd = "docker-compose -f '%s' config" % (compose_file_name)
+    try:
+        run(cmd)
+    except Exception:
+        return 'docker-compose file is not valid'
```

Comment:
Should we also log a `LOG.warning` here (or even raise an Exception), instead of returning the string?
```
    except Exception as e:
        LOG.warning('Looks like the docker-compose file is not valid: %s' % e)
        return False
```